src/components/MultiSelect/MultiSelect.mdx

```diff
@@ -2,18 +2,22 @@ import { ArgTypes, Canvas, Description, Meta, Source } from "@storybook/blocks";
 
 import Banner from "../Banner/Banner";
 import ComponentChangelogTable from "../../utils/ComponentChangelogTable";
-import * as MultiSelectStories from "./MultiSelect.stories";
+import ComponentDocsHeader from "../../utils/components/ComponentDocsHeader";
 import Link from "../Link/Link";
+import * as MultiSelectStories from "./MultiSelect.stories";
 import { changelogData } from "./multiSelectChangelogData";
 
 <Meta of={MultiSelectStories} />
 
-# MultiSelect
+<ComponentDocsHeader
+  category="Form elements"
+  componentName="MultiSelect"
+  summary="Grouping of related options rendered in a menu format that allows a user to select one or more options"
```

Comment:
I think this could also be updated to checkboxes, but if you feel best about "options" I think that's fine.

"Grouping of related checkboxes rendered in a menu format that allows a user to select one or more options"